src/Components/NotificationItem.vue

```diff
@@ -104,6 +105,34 @@ import FileParameter from './Parameters/FileParameter.vue'
 import UserParameter from './Parameters/UserParameter.vue'
 import { formatDateTime, formatRelativeTimeFromNow } from '../utils/datetime.js'
 
+/**
+ * @typedef {object} NotificationItem
```

Comment:
this is duplicate?

Reply:
This actually doesn't belong to this PR, and ActionButton component in particular. Click method should be moved in NotificationItem